AMaDda/Model/UserDefaultsStateManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FamilyContactManager.swift
+//  AMaDda
+//
+//  Created by Lee Myeonghwan on 2022/07/20.
+//
+
+import Foundation
+
+final class UserDefaultsStateManager {
+    private var today: Date {
+        return convertKoreaDate(Date())
+    }
+    func userEnteredApp() {
+        guard var finalEnteredDate = UserDefaults.standard.finalEnteredDate else {
+            UserDefaults.standard.finalEnteredDate = today
+            return
+        }
+        finalEnteredDate = convertKoreaDate(finalEnteredDate)
+        let offsetDateComponents = Calendar.current.dateComponents([.day], from: finalEnteredDate, to: today)
+        guard let offsetDay = offsetDateComponents.day else { return }
+        if offsetDay == 0 {
+            return
+        } else {
+            updateFinalContactDiffDay(finalEnteredDate)
+            UserDefaults.standard.finalEnteredDate = today
+        }
+    }
+    private func updateFinalContactDiffDay(_ finalEnteredDate: Date) {
+        guard var finalContactDiffDay = UserDefaults.standard.finalContactDiffDay else { return }
+        let offsetDateComponents = Calendar.current.dateComponents([.day], from: finalEnteredDate, to: today)
+        guard let offsetDay = offsetDateComponents.day else { return }
+        finalContactDiffDay += offsetDay
+        UserDefaults.standard.finalContactDiffDay = finalContactDiffDay
+    }
+    private func convertKoreaDate(_ convertDate: Date) -> Date {
+        let hourAsSecond: Int = 3600
+        let koreaGreenwichDiff = TimeInterval(hourAsSecond*9)
```

Comment:
```suggestion
        let koreaGreenwichDiff = TimeInterval(hourAsSecond * 9)
```

p3;

연산자 사이에 띄어쓰기를 추가하면 피연산자가 구분이 잘 돼서 가독성이 좋아질 거 같아요~

Reply:
놓친 부분이네요 감사합니다~!